src/bun.js/api/server/NodeHTTPResponse.zig

```diff
@@ -269,7 +269,9 @@ pub fn create(
     node_response_ptr: *?*NodeHTTPResponse,
 ) callconv(.C) JSC.JSValue {
     const vm = globalObject.bunVM();
-    if ((HTTP.Method.which(request.method()) orelse HTTP.Method.OPTIONS).hasRequestBody()) {
+    const method = HTTP.Method.which(request.method()) orelse HTTP.Method.OPTIONS;
+    // GET in node.js can have a body
```

Comment:
This is also for [RFC72731](https://datatracker.ietf.org/doc/html/rfc7231)

> To be consistent with the method-neutral parsing algorithm of
   [[RFC7230](https://datatracker.ietf.org/doc/html/rfc7230)], the definition of GET has been relaxed so that requests
   can have a body, even though a body has no meaning for GET.